EventFilter/GEMRawToDigi/plugins/GEMRawToDigiModule.cc

```diff
@@ -100,7 +115,13 @@ std::shared_ptr<GEMROMapping> GEMRawToDigiModule::globalBeginRun(edm::Run const&
 
 void GEMRawToDigiModule::produce(edm::StreamID iID, edm::Event& iEvent, edm::EventSetup const& iSetup) const {
   auto outGEMDigis = std::make_unique<GEMDigiCollection>();
-  auto outVFATStatus = std::make_unique<GEMVfatStatusDigiCollection>();
+  auto outAMC13Status = std::make_unique<GEMAMC13StatusCollection>();
+  auto outAMCStatus = std::make_unique<GEMAMCStatusCollection>();
+  auto outOHStatus = std::make_unique<GEMOHStatusCollection>();
+  auto outVFATStatus = std::make_unique<GEMVFATStatusCollection>();
+
+  // to be removed
```

Comment:
could this comment be clarified? to be removed by whom, when, which github issue?

Reply:
@jpata I will remove once the onlineDQM has been updated to no longer use those collections